<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StewartPlatformClass</class>
 <widget class="QMainWindow" name="StewartPlatformClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StewartPlatformSimulator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QOpenGLWidget" name="renderer" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_CL">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>30</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Cylinders Lengths</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_POS">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>260</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Posture</string>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>290</y>
      <width>141</width>
      <height>152</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_x">
       <property name="text">
        <string>x (m)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_y">
       <property name="text">
        <string>y (m)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_z">
       <property name="text">
        <string>z (m)</string>
       </property>
       <property name="buddy">
        <cstring>text_body_yR</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_zR">
       <property name="text">
        <string>zR (deg)</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_yR">
       <property name="text">
        <string>yR (deg)</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_xR">
       <property name="text">
        <string>xR (deg)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="text_body_y">
       <property name="text">
        <string>text_body_y</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="text_body_z">
       <property name="text">
        <string>text_body_z</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="text_body_zR">
       <property name="text">
        <string>text_body_zR</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="text_body_xR">
       <property name="text">
        <string>text_body_xR</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="text_body_yR">
       <property name="text">
        <string>text_body_yR</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="text_body_x">
       <property name="text">
        <string>text_body_x</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>290</y>
      <width>161</width>
      <height>152</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_angleP">
       <property name="text">
        <string>angleP (deg)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_angleB">
       <property name="text">
        <string>angleB (deg)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="text_body_angleB">
       <property name="text">
        <string>text_body_angleB</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_radiusP">
       <property name="text">
        <string>radiusP (mm)</string>
       </property>
       <property name="buddy">
        <cstring>text_body_yR</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="text_body_radiusP">
       <property name="text">
        <string>text_body_radiusP</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_radiusB">
       <property name="text">
        <string>radiusB (mm)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="text_body_radiusB">
       <property name="text">
        <string>text_body_radiusB</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_Lmin">
       <property name="text">
        <string>Lmin (mm)</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="text_body_Lmin">
       <property name="text">
        <string>text_body_Lmin</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_stroke">
       <property name="text">
        <string>stroke (mm)</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="text_body_stroke">
       <property name="text">
        <string>text_body_stroke</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="text_body_angleP">
       <property name="text">
        <string>text_body_angleP</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_PP">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>260</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Platform Parameters</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>60</y>
      <width>371</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,3,8">
     <property name="horizontalSpacing">
      <number>15</number>
     </property>
     <item row="0" column="1">
      <widget class="QLabel" name="label_L1">
       <property name="text">
        <string>L1 (mm)</string>
       </property>
       <property name="buddy">
        <cstring>SliderCL1</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_L4">
       <property name="text">
        <string>L4 (mm)</string>
       </property>
       <property name="buddy">
        <cstring>SliderCL4</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_L2">
       <property name="text">
        <string>L2 (mm)</string>
       </property>
       <property name="buddy">
        <cstring>SliderCL2</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_L3">
       <property name="text">
        <string>L3 (mm)</string>
       </property>
       <property name="buddy">
        <cstring>SliderCL3</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="text_body_L2">
       <property name="text">
        <string>text_body_L2</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QSlider" name="SliderCL2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="text_body_L1">
       <property name="text">
        <string>text_body_L1</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QSlider" name="SliderCL1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="text_body_L3">
       <property name="text">
        <string>text_body_L3</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QSlider" name="SliderCL3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_L5">
       <property name="text">
        <string>L5 (mm)</string>
       </property>
       <property name="buddy">
        <cstring>SliderCL5</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="label_L6">
       <property name="text">
        <string>L6 (mm)</string>
       </property>
       <property name="buddy">
        <cstring>SliderCL6</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="text_body_L4">
       <property name="text">
        <string>text_body_L4</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLineEdit" name="text_body_L5">
       <property name="text">
        <string>text_body_L5</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLineEdit" name="text_body_L6">
       <property name="text">
        <string>text_body_L6</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QSlider" name="SliderCL4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QSlider" name="SliderCL5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QSlider" name="SliderCL6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>490</y>
      <width>47</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QOpenGLWidget</class>
   <extends>QWidget</extends>
   <header>qopenglwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="StewartPlatformSimulator.qrc"/>
 </resources>
 <connections/>
</ui>
